strings/tips.txt

```diff
@@ -0,0 +1,205 @@
+Where the space map levels connect is randomized every round, but are otherwise kept consistent within rounds. Remember that they are not necessarily bidirectional!
```

Comment:
It's unclear what bidrectional means here, so maybe it might be confusing to put it here. Something to leave on the wiki, maybe? Or maybe I'm just confused 😕

Reply:
Fixed in next commit